implementations/rust/ockam/ockam_entity/src/credential/macros.rs

```diff
@@ -12,8 +12,12 @@ macro_rules! credential_type {
         use $crate::CredentialAttributeSchema;
         use $crate::CredentialAttributeType::{Number, Utf8String, Blob};
 
-        #[allow(unused_mut)]
-        let mut attributes = vec![];
+        let mut attributes = vec![CredentialAttributeSchema {
+            label: $crate::SECRET_ID.into(),
+            description: "".to_string(),
+            attribute_type: Blob,
+            unknown: true,
+        }]; // FIXME
```

Comment:
For FIXMEs would you mind also adding a very short description, for clarity? In this case it is that the schema should not be constructed but passed in from somewhere?